project/Dependencies.scala

```diff
@@ -82,11 +82,12 @@ object Dependencies {
     val akkaStreamTestKit = "com.typesafe.akka" %% "akka-stream-testkit" % akkaVersion % "test"
     val junit = "junit" % "junit" % "4.12" % "test"
     val junitInterface = "com.novocode" % "junit-interface" % "0.10" % "test"
-    val spark = Compile.spark.map(_ % "test")
+    val spark = Compile.spark.map(_ % "test") ++ Compile.spark.map(_ % "test" classifier "tests")
   }
 
   object Provided {
     val spark = Compile.spark.map(_.excludeAll(ExclusionRule(organization = "org.scalatest"))).map(_ % "provided")
+    val sparkTestLib = spark.map(_ classifier "tests")
```

Comment:
We don't need test jars in runtime. So they shouldn't under "Provided".

Reply:
Oh, this is a special case, referenced by sub-project mleap-spark-testkit (this project has no test code, the main src code need reference spark testing API)